django/contrib/auth/forms.py

```diff
@@ -141,6 +141,8 @@ def save(self, commit=True):
         user.set_password(self.cleaned_data["password1"])
         if commit:
             user.save()
+            if hasattr(self, "save_m2m"):
+                self.save_m2m()
```

Comment:
I think we should add a `versionchanged` annotation to the docs:
- [`UserCreationForm` docs](https://docs.djangoproject.com/en/dev/topics/auth/default/#django.contrib.auth.forms.UserCreationForm),
- ["Custom users and the built-in auth forms" docs](https://docs.djangoproject.com/en/dev/topics/auth/customizing/#custom-users-and-the-built-in-auth-forms),

and release notes.

Reply:
Done!